tests/zfs-tests/tests/functional/chattr/chattr_004_neg.ksh

```diff
@@ -0,0 +1,103 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or https://opensource.org/licenses/CDDL-1.0.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+
+#
+# Copyright (c) 2013 by Delphix. All rights reserved.
+# Copyright (c) 2017 by Fan Yong. All rights reserved.
+# Copyright 2022 Zettabyte Software, LLC.  All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+#	Verify FS_CASEFOLD_FL is unsupported on case sensitive filesystems.
+#
+# STRATEGY:
+#	1. Create a filesystem with casesensitivity=insensitive.
```

Comment:
```suggestion
#	1. Create a filesystem with casesensitivity=sensitive.
```

Reply:
Nice find. Thanks. I just amended my commit locally. I will repush after posting this comment.